src/loader.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef _LOADER_HPP_
+#define _LOADER_HPP_
+
+#include "datatype.hpp"
+#include <fstream>
+#include <iostream>
```

Comment:
`#include <fstream>` and `#include <iostream>` should be added in `datatype.hpp`